docs/interactive-scroll-view.md

```diff
@@ -0,0 +1,40 @@
+---
+id: interactive-scroll-view
+title: ScrollView 
+sidebar_label: ScrollView
+---
+
+Render `children` elements in a scrollable container with fixed height, using mouse scroll events. Useful for reporters and logs display.
+
+## Props
+
+* `children: React.Element<*>` - Content to be rendered and scrolled.
+* `height: number` - Fixed height of the visible area.
+* `inputStream: tty$ReadStream` (default: `process.stdin`) - Input stream to capture scroll events from.
+* `outputStream: tty$WriteStream` (default: `process.stdout`) - Output stream to enable/disable mouse reporting, hence toggling scroll events.
+* `reportingRatio: number` (default: `0.25`) - Mouse scrolling reporting ratio, effectively controlling the speed of scroll.
+* `inverted?: boolean` - Inverts the direction of scrolling.
+* `style?: Style` - Object with [Style properties](./core-style-prop.md).
+* `follow?: boolean`, - Makes `ScrollView` follow new content added at the bottom.
```

Comment:
What does this do?

Reply:
afaik, basically, if the View is scrolled to the bottom this property will force the ScrollView to keep scrolling with a new content. Similar to `tail -f`.

Reply:
Maybe `anchorToBottom` or something? `follow` isn't very clear on what it does.

Reply:
`anchorToBottom` is even more cryptic. `follow` follows new content added to the bottom, it automatically scrolls the container to always show the latest element.

Reply:
`stickToBottom`? Or `followContent`. I'm cool with `follow` too